app/assets/stylesheets/components/_memorable-date.scss

```
@@ -0,0 +1,4 @@
+.input-padding {
+    padding-right: 0.5rem !important;
+    padding-left: 0.5rem !important;
+}
```

`!important` should ideally never be used. It looks like it's not working without `!important` because of [this selector](https://github.com/18F/identity-idp/blob/3fd5c3bbe72cb32f672a39b030eb3b65a8dc09fc/app/assets/stylesheets/design-system-waiting-room.scss#L5-L26) which has the same specificity and is defined after these styles. We can make these styles win by forming the selector in a way that targets those styles and overrides them. (Ideally we'll be able to remove those styles at some point anyways).

Also, we should use [design system spacing units](https://designsystem.digital.gov/design-tokens/spacing-units/) rather than specific REM values.

```suggestion
input[type='text'].memorable-date {
  @include u-padding-x(1);
}
```

---

Updated the style to use the design system units.